cmd/volume.go

```diff
@@ -122,20 +208,21 @@ func volumeAttachCommand() *cobra.Command {
 
 // VolumeCommands handles volumes related operations
 func VolumeCommands() *cobra.Command {
-	var config string
-	var provider string
+	var config, label, provider string
 	var nightly bool
 	cmdVolume := &cobra.Command{
 		Use:       "volume",
 		Short:     "manage nanos volumes",
-		ValidArgs: []string{"create, list, delete, attach"},
+		ValidArgs: []string{"create, list, update, delete, attach"},
 		Args:      cobra.OnlyValidArgs,
 	}
 	cmdVolume.PersistentFlags().StringVarP(&config, "config", "c", "", "ops config file")
-	cmdVolume.PersistentFlags().StringVarP(&provider, "provider", "p", "", "cloud provider")
+	cmdVolume.PersistentFlags().StringVarP(&label, "label", "l", api.DefaultVolumeLabel, "volume label")
+	cmdVolume.PersistentFlags().StringVarP(&provider, "provider", "p", "onprem", "cloud provider")
```

Comment:
i'm not married to the short flags but for now the convention looks like '-t' instead of '-p' in most places

Reply:
updated the flag